tests/fast/VersionStamp.txt

```diff
@@ -1,2 +1,3 @@
 testTitle=VersionStamp
-testName=VersionStamp
\ No newline at end of file
+testName=VersionStamp
+onlyTestChangingMetadataVersion=true
```

Comment:
Did you intend to leave this line in permanently?

Reply:
I now realize the name is confusing. It is supposed to mean it is the only test changing this variable, so it has sole ownership of the value and can do more strict validation.

I now realize it sounds like this option is weakening the test instead of strengthen it